web.md

```diff
@@ -0,0 +1,47 @@
+# Web
+
+**Prerequisite knowledge**: null
+
+No prerequisite knowledge required. If you can see this document, you are probably aware of what computers are and that they can communicate with each other via global system of interconnected computer networks, called the Internet. If so, you're all set to begin your journey with the web.
+
+## Intro
+Web - short for World Wide Web.
+
+## Classification / acceptance questions
+### Apprentice aka _I know what Web is_:
+- I can explain how does the Web work and what are its most fundamental building blocks.
+- I can tell the difference between the Web and the Internet.
+- I know what a web browser is.
+- I know what a web server is.
+
+**Where to look for answers**:
+- https://en.wikipedia.org/wiki/World_Wide_Web
+- https://www.youtube.com/watch?v=J8hzJxb0rpc
+
+### Intermediate aka _I know what's going on in the web_
+- I can explain how does the Web work.
+- I know what DNS, HTTP and HTML are.
+- I can describe step by step what happens when I put `www.youtube.com` into the address bar and hit enter before I can see the results. (Level of detail really depends on You. The more specific the answer, the better. But let's say naming 5 - 7 key steps  is enough to pass that question. Describing what exactly happens on youtube servers would be an overkill :D)
+
+**Prerequisite knowledge**: DNS, HTTP/HTTPS, HTML
```

Comment:
I think it's important to know what is required from you before you start answering questions. Maybe we could move this line directly under the level name?